themes/default/layouts/shortcodes/install-java.html

```diff
@@ -2,3 +2,15 @@
     Install <a href="https://www.oracle.com/java/technologies/downloads" target="_blank">Java 11 or later</a>
     and <a href="https://maven.apache.org/install.html">Apache Maven 3.5 or later</a>.
 </p>
+
+<div class="note note-info" role="alert">
+    <div class="note-heading">
+        <i class="fas fa-info-circle mr-1"></i>
+        <span>Note</span>
+    </div>
+    <div class="note-body">
+        <p>
+            Pulumi will need the <code>java</code>, <code>javac</code>, and <code>mvn</code> executables in order to build and run your Pulumi Java application. Ensure that these executables can be found on your path after installation.
```

Comment:
Would it be worth linking to something like sdkman here?

Reply:
Interesting idea, yeah. I was mainly just looking to add some details for Java (and for consistency with the other languages), but it raises a good point -- given how common and useful these kinds of version managers are now, it'd probably be nice if we had page, or a section for each page under Intro > Languages, specifically for making these kinds of recommendations. What do you think? I'll file an issue if it sounds like something useful.